recipe/meta.yaml

```diff
@@ -1,105 +1,117 @@
+{% set vendored_libs = ["address_sorting"] %}
+{% set binaries_plugin_langs = [
+    "cpp", "csharp", "node", "objective_c", "php", "python", "ruby"
+] %}
 
 package:
-  name: {{ name | lower }}-suite
+  name: grpc-split
   version: {{ version }}
 
 source:
-  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: bdb8e98145469d58c69ab9f2c9e0bd838c2836a99b5760bc0ebf658623768f52
+  url: https://github.com/grpc/grpc/archive/v{{ version.replace(".pre", "-pre") }}.tar.gz
+  sha256: e5d5e0dd96fe9452fe24cc8c827381dca484c54d171fb512a198025fec81a3c8
   patches:
     - patches/0001-windows-ssl-lib-names.patch
     - patches/0002-fix-win-setup-cmds.patch
     - patches/0003-Link-against-grpc.patch
     - patches/0004-force-protoc-executable.patch
-    - patches/0005-patch-tests-to-use-passed-CXX_STANDARD.patch
-    - patches/0006-switch-to-C-17-for-grpcio.patch
-    - patches/0007-fix-abseil-setup.patch
+    - patches/0005-switch-to-C-17-for-grpcio.patch
+    - patches/0006-fix-abseil-setup.patch
+    # mark c-ares, re2 & zlib as private, so they don't need to be found
+    # at runtime just to link to grp, see grpc/grpc#30838;
+    # not much point in marking baselibs or vendored libs (e.g. upb) as private
+    - patches/0007-mark-linkage-of-c-ares-re2-zlib-as-private.patch
+    - patches/0008-link-grpc-_unsecure-to-grpc.patch
+    # see https://github.com/grpc/grpc/issues/35794
+    - patches/0009-fix-missing-DLL-exports-in-vendored-upb.patch
+    # fix symbol visibility regression on windows
+    - patches/0010-for-main-targets-set-_DLL_EXPORTS-when-building-and-.patch
 
 build:
-  number: 4
-  #skip: true  # [not win]
+  number: 0
+
+requirements:
+  build:
+    - patch # [unix]
+    - m2-patch # [win]
 
 outputs:
-  - name: {{ name }}-cpp
-    script: build-cpp.sh   # [not win]
+  - name: libgrpc
+    script: build-cpp.sh  # [not win]
     script: build-cpp.bat  # [win]
     build:
-      missing_dso_whitelist:  # [s390x]
-        - '$RPATH/ld64.so.1'  # [s390x]
-      script_env:
-        - LIB_EXT={{libext}}
       run_exports:
-        - {{ pin_subpackage('grpc-cpp', max_pin='x.x') }}
+        - {{ pin_subpackage('libgrpc', max_pin='x.x') }}
+        # grpc requires C11 aligned_alloc as of 1.56;
+        # ensure that all dependent packages reflect that to
+        # avoid the solver getting stuck in no-man's-land.
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx and x86_64]
+      missing_dso_whitelist:  # [s390x]
+        - $RPATH/ld64.so.1    # [s390x]
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - m2-patch  # [win]
-        - patch  # [not win]
-        - cmake >=3.5.1
-        - pkg-config  # [unix]
-        # `protoc` is also used for building
-        - libprotobuf {{ libprotobuf }}
-        - ninja
+        - cmake
+        - ninja-base
+        # During cross-compilation, we need to build the grpc_cpp_plugin for the
```

Comment:
maybe I'm missing something, but when do we do cross compilation?

Reply:
We don't, but most changes here are synced from conda-forge.